src/components/card/Card/Card.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import classnames from 'classnames'
+
+interface CardProps {
+  header?: React.ReactNode
+  media?: React.ReactNode
+  footer?: React.ReactNode
+  standardLayout?: 'default' | 'headerFirst'
+  flagLayout?: 'default' | 'mediaOnRight'
+}
+
+export const Card = (
+  props: CardProps & React.HTMLAttributes<HTMLDivElement>
+): React.ReactElement => {
+  const {
+    header,
+    media,
+    footer,
+    standardLayout,
+    flagLayout,
+    children,
+    className,
+    ...divProps
+  } = props
+
+  // TODO determine how to implement the tablet:grid-col-4 class properly, should this be li class?
```

Comment:
Assuming the `li` can handle any of the various responsive grid classes, maybe this component can share the grid prop structure and class helper function located in https://github.com/trussworks/react-uswds/blob/develop/src/components/grid/Grid/Grid.tsx#L11 ? That could be extracted out and shared across any components that support grid classes.

Reply:
I am not quite sure how to do this? Can you explain further?

Reply:
Paired with @suzubara to implement.